Docs/userguide/on_target_inference.rst

```diff
@@ -6,13 +6,13 @@
 On-target inference
 ###################
 
-In order to run AIMET quantized model on a target device, you need following two things:
+Running an AIMET quantized model on a target device requires two things:
 
-- an exported model,
```

Comment:
Is the exported model a single file or a collection of files? What are the file extensions?